### PR TITLE
Notes: load shared note based on the url parameter

### PR DESCRIPTION
**Pull request by OlegMoshkovich**
This PR is related to #314.
SideDrawer useEffect clears the issue number and moves the navpanel to the list view. .

### CODE REVIEW

**Comment by pablo-mayrgundter** [comment]
Is this description correct?  Not sure what this is doing.. isn't it handling clearing behavior?

**Comment by pablo-mayrgundter** [comment]
Is this description correct?  Not sure what this is doing.. isn't it handling clearing behavior?

File: src/Components/SideDrawer.jsx [hunk 1]
```diff
@@ -101,7 +101,11 @@ export default function SideDrawerWrapper() {
       openDrawer()
       turnCommentsOn()
     }
-  }, [location, openDrawer, setSelectedIssueId, turnCommentsOn])
+    // This address bug #314 by clearing selected issue when new model is loaded
+    if (issueHash === undefined && isDrawerOpen) {
+      setSelectedIssueId(null)
```
Comment (OlegMoshkovich) [inline src/Components/SideDrawer.jsx:106]: it clears the selected note and moves the navpanel to the list view, on model reload.

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter PTAL.

**Comment by pablo-mayrgundter** [comment]
Hi, can you merge and now try a test for this?

File: src/Components/SideDrawer.test.jsx [hunk 2]
```diff
@@ -80,3 +80,23 @@ test('side drawer - opened via URL', async () => {
   })
 })
 
+
+test('Shared issue with comments via URL - test for issue body', async () => {
+  const {result} = renderHook(() => useStore((state) => state))
```
Comment (OlegMoshkovich) [inline src/Components/SideDrawer.test.jsx:85]: @pablo-mayrgundter the test is here.

**Review by pablo-mayrgundter — APPROVED** [review]

**Comment by pablo-mayrgundter** [comment]
Heya, let's try to avoid significant mods after Approval without another round of review.  What ended up happening?  Looks like the test for this got removed?

**Comment by OlegMoshkovich** [comment]
I thought you approved it.

**Comment by pablo-mayrgundter** [comment]
Per discussion, please restore test in a new PR.  Thanks!

**Pull request merged**